man/sd_varlink_send.xml

```diff
@@ -0,0 +1,149 @@
+    <refname>sd_varlink_sendb</refname>
+    <refname>sd_varlink_sendbo</refname>
+
+    <refpurpose>Enqueues a Varlink method call, not expecting a reply</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-varlink.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int <function>sd_varlink_send</function></funcdef>
+        <paramdef>sd_varlink *<parameter>link</parameter></paramdef>
+        <paramdef>const char *<parameter>method</parameter></paramdef>
+        <paramdef>sd_json_variant *<parameter>parameters</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_varlink_sendb</function></funcdef>
+        <paramdef>sd_varlink *<parameter>link</parameter></paramdef>
+        <paramdef>const char *<parameter>method</parameter></paramdef>
+        <paramdef>…</paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>sd_varlink_sendbo</function></funcdef>
+        <paramdef>sd_varlink *<parameter>link</parameter></paramdef>
+        <paramdef>const char *<parameter>method</parameter></paramdef>
+        <paramdef>…</paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_varlink_send()</function> submits a method call via a Varlink connection. It takes the
+    Varlink connection object, a method name as string parameter, and a JSON object containing the parameters
+    to pass as function parameters. This call is asynchronous: the message will not be delivered immediately
+    but only once
+    <citerefentry><refentrytitle>sd_varlink_process</refentrytitle><manvolnum>3</manvolnum></citerefentry> is
+    invoked (which will happen automatically in one of the following event loop iterations if the Varlink
+    connection is attached to an even loop).</para>
+
+    <para><function>sd_varlink_sendb()</function> is similar to <function>sd_varlink_send()</function>, but
+    instead of expecting a fully constructed <type>sd_json_variant</type> object carrying the parameters,
+    this object is constructed on-the-fly directly from the parameter list, in a style identical to
+    <citerefentry><refentrytitle>sd_json_build</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
+
+    <para><function>sd_varlink_sendbo()</function> is identical to <function>sd_varlink_sendb()</function>,
+    but an enclosing object is implicitly added, so that the parameter list is expected to consist of field
+    pairs only. For details about the expected argument list, see
+    <citerefentry><refentrytitle>sd_json_buildo</refentrytitle><manvolnum>3</manvolnum></citerefentry>.</para>
+
+    <para>Use
+    <citerefentry><refentrytitle>sd_json_call</refentrytitle><manvolnum>3</manvolnum></citerefentry> (and
+    related calls) to submit a method call returning the server's response.</para>
```

Comment:
I cannot find why the paragraph exists here. What do you want to explain here?

Reply:
sd_varlink_send() is a fire-and-forget operation, i.e. just issues the method call, and provides no mechanism to get a reply. sd_varlink_call() is hence the typically the call people want, since it is the synchronous operation people usually expect, hence I wanted to add a cross reference to it.

(of course sd_varlink_call() is currently documented, but sooner or later we should fill in that gap)

Reply:
Thanks. Then, please extend the paragraph to say that.